listenbrainz/db/feedback.py

```diff
@@ -22,15 +23,15 @@ def insert(feedback: Feedback):
         connection.execute(sqlalchemy.text("""
             INSERT INTO recording_feedback (user_id, recording_msid, score)
                  VALUES (:user_id, :recording_msid, :score)
-            ON CONFLICT (user_id, recording_msid)
+            ON CONFLICT (user_id, recording_msid, recording_mbid)
```

Comment:
The issue with this is that recording_mbid are currently all NULL so we'll get duplicate feedbacks. However, we end up in the same situation if we insert using just recording_mbid because we already have msid feedback. One way could be to run a script to insert mbids for each msid.